docs/osf/setup.rst

```diff
@@ -336,24 +369,26 @@ The following commands will install the requirements for add ons.
         invoke addon_requirements
 
 
-Starting up
------------
+Running The OSF
+***************
 
-Run your TokuMX process:
+Quick Start
+-----------
+To fully run the OSF, the following commands must be run.  Many of these programs will continue to run in order to log output to the console: this is normal!
 
+Therefore, run the following commands each in their own terminal windows, making sure to switch to the OSF virtual environment (and directory) each time:
     ::
 
-        invoke mongo
-
-Note -- TokuMX must be running in order to invoke the server. If the process stops it has failed. Try running  `mongod` for a more informative message. See below for common problems.
+            invoke mongo -d  # Runs mongod as a daemon
```

Comment:
We can probably remove `-d`. We don't daemonize any of the other processes, so mongo doesn't have to be any different.